src/azure-cli/azure/cli/command_modules/role/custom.py

```diff
@@ -1025,6 +1032,29 @@ def _build_app_roles(app_roles):
     return result
 
 
+def _build_optional_claims(optional_claims):
+    if not optional_claims:
+        return None
+    available_keys = {
+        'idToken': 'id_token',
+        'accessToken': 'access_token'
+    }
+    result = OptionalClaims()
+    for key, object_key in available_keys.items():
+        if optional_claims.get(key, []):
```

Comment:
since no schema check on json content, what if input optional_claims doesn't contain either idToken or accessToken?

Reply:
It will skip these properties if it is not in the json file. It's because none of these properties is required.